src/language/forms.rkt

```diff
@@ -303,6 +302,8 @@ And a type constructor "name" expecting the types of field1 and field2
   #:transparent)
 (struct Assoc-Stack base-type ()
   #:transparent)
+(struct Suspended-Cast base-type ()
```

Comment:
Would it be more appropriate to call this a suspended cast queue?

Reply:
This is dead code, I deleted it.